src/PKSim.Core/PKSim.Core.csproj

```diff
@@ -107,6 +107,9 @@
       <RequiredTargetFramework>3.5</RequiredTargetFramework>
     </Reference>
     <Reference Include="System.Drawing" />
+    <Reference Include="System.ValueTuple, Version=4.0.1.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51, processorArchitecture=MSIL">
```

Comment:
This is a neat package until we move to .NET 4.7 .We can use tuple as return value 
sthg like 
```
(string a, string b) function test(){
}
```

I did not use it after all, but we kept it anyways for later use